src/main/g8/build.sbt

```diff
@@ -1,15 +1,17 @@
-lazy val akkaHttpVersion = "$akka_http_version$"
-lazy val akkaVersion    = "$akka_version$"
+lazy val akkaHttpVersion = "10.0.9"
+lazy val akkaVersion    = "2.5.3"
 
 lazy val root = (project in file(".")).
   settings(
     inThisBuild(List(
       organization    := "com.example",
-      scalaVersion    := "$scala_version$"
+      scalaVersion    := "2.12.2"
```

Comment:
As this is now fixed to Scala 2.12.2, `travis.yml` can be configured accordingly so it only builds for this version and not for both Scala 2.11.11 and 2.12.2